src/constants/chainTokens.js

```diff
@@ -534,6 +534,12 @@ export const chainCoingeckoIds = {
     symbol: "BOBA",
     cmcId: "14556",
   },
+  Candle: {
+    geckoId: "candle",
+    symbol: "CNDL",
+    cmcId: "18227",
+
```

Comment:
Hey @samisbakedham

I think you could remove this space.

Reply:
Fixed!